chirpr.py

```diff
@@ -83,9 +86,15 @@ def login_page():
 def user_page(handle):
     uid = db_access.get_id(handle)
     my_followers = []
+    name = "Stranger"
+    chirp_list = []
     if 'user' in session:
         my_followers = get_followers()
-    return render_template('user_page.html', handle=handle, uid=uid, follow_data=db_access.follow_data(uid), my_followers=my_followers, get_user=db_access.get_user)
+        user = session['user']
+        name = db_access.get_user(user)
+        if db_access.follower_of(uid, user):
+            chirp_list = db_access.get_chirps(uid)
+    return render_template('user_page.html', handle=handle, uid=uid, follow_data=db_access.follow_data(uid), my_followers=my_followers, get_user=db_access.get_user,chirps=chirp_list, name=name)
```

Comment:
I think the `get_chirps` function should work for all users, whether they are following someone or not.

It should maybe be called `get_chirps_for_timeline` or something that indicates that these are not the chirps of this user.

I'm also curious -- I'm sure I'll figure it out in a minute -- what `follow_data` is. That is a very non-descriptive name.